inspirehep/modules/workflows/tasks/classifier.py

```diff
@@ -38,6 +40,74 @@
 from ..utils import with_debug_logging, get_document_in_workflow
 
 
+CLASSIFIER_MAPPING = {
+    "core": "CORE",
+    "non_core": "Non-CORE",
+    "rejected": "Rejected"
+}
+
+
+def get_classifier_url():
+    """Return the classifier URL endpoint, if any."""
+    base_url = current_app.config.get('CLASSIFIER_API_URL')
+    if not base_url:
+        return None
+
+    return '{base_url}/predict/coreness'.format(base_url=base_url)
+
+
+def prepare_payload(record):
+    """Prepare payload to send to Inspire Classifier."""
+    payload = {
+        'title': get_value(record, 'titles.title[0]', ''),
+        'abstract': get_value(record, 'abstracts.value[0]', ''),
+    }
+    return payload
+
+
+@with_debug_logging
+def guess_coreness(obj, eng):
+    """Workflow task to ask inspire classifier for a coreness assessment."""
+    predictor_url = get_classifier_url()
+    if not predictor_url:
+        return None
```

Comment:
just `return` as this is used for control flow only, the function never returns anything.